lessons/13-css-external-files/about.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>About</title>
+  </head>
+
+  <header>
+    <h1>About</h1>
+    <link rel="stylesheet" href="style.css" />
+    <nav>
+      <a href="index.html">Home</a>
+      <a class="change-color" href="about.html">About</a>
+      <a href="news.html">News</a>
+    </nav>
+  </header>
```

Comment:
And this needs to go inside of the `<body>` since it is being displayed to the user